### ISSUE TITLE
Add Windows version

### ISSUE THREAD

**Issue by Pzoom522** [mozillazg/ptcpdump#210]
Thanks for releasing this beautiful project! It is possible to support Windows OS as well?

**Comment by mozillazg** [comment]
@Pzoom522 Yes, It's possible to support Windows based on https://github.com/microsoft/ebpf-for-windows. I'll implement this feature in the future.

**Comment by Pzoom522** [comment]
> @Pzoom522 Yes, It's possible to support Windows based on https://github.com/microsoft/ebpf-for-windows. I'll implement this feature in the future.

Looking forward to it! May I know the planned date/time? Thx!

**Comment by mozillazg** [comment]
> Looking forward to it! May I know the planned date/time? Thx!

There is no clear date for this plan yet.

**Comment by heheda123123** [comment]
支持，很有用
